ebean-core/src/test/java/org/tests/query/cancel/SqlQueryCancelTest.java

```diff
@@ -113,8 +113,8 @@ public void cancelOrmQueryAtBegin() throws SQLException {
     doCancelOrmAtBegin(Query::findOne);
     doCancelOrmAtBegin(q -> q.setMaxRows(1000).findPagedList().getList()); // untested
     doCancelOrmAtBegin(Query::findSet);
-    doCancelOrmAtBegin(Query::findSingleAttribute);
-    doCancelOrmAtBegin(Query::findSingleAttributeList);
+    doCancelOrmAtBegin(q -> q.select("name").findSingleAttribute());
+    doCancelOrmAtBegin(q -> q.select("name").findSingleAttributeList());
```

Comment:
specifying an attribute is mandatory now, otherwise you'll get an assertionError `selectSql was null`